exonum/src/node/mod.rs

```diff
@@ -439,7 +474,21 @@ impl NodeHandler {
             system_state.current_time(),
         );
 
-        let is_enabled = api_state.clone().is_enabled();
+        let mut is_enabled = api_state.clone().is_enabled();
+        let node_role = match validator_id {
```

Comment:
This match is used three times in code, what about moving it in  a method?

Reply:
Yeah, it should be better.